spec/support/features/idv_helper.rb

```diff
@@ -123,7 +121,6 @@ def complete_idv_profile_ok(user, password = user_password)
     fill_out_idv_form_ok
     click_idv_continue
     click_idv_address_choose_phone
-    fill_out_phone_form_ok(user.phone)
```

Comment:
The user's phone is already prepopulated. No need to explicitly fill it in again. In addition, bug LG-213 prevents us from filling the user's phone because it contains the country code by default. We could change the user's phone to omit the country code, but since the phone number is already filled in, might as well make the test faster.